### PR TITLE
vSRX fixes and improvements

### PR DESCRIPTION
**Pull request by ssasso**
Cherrypicking and superseding #118 and #119

* Improved commit phase (it seems that sometimes the "commit and-quit" is stuck)
* Management VRF for fxp0 traffic (and default route for it)

(@hellt unable to do a direct fork from #118, so I did a cherrypick from both #118 and #119)

### CODE REVIEW

File: vsrx/docker/launch.py [hunk 1]
```diff
@@ -102,8 +102,15 @@ def bootstrap_config(self):
         self.wait_write(self.password, "New password:")
         self.wait_write(self.password, "Retype new password:")
         self.wait_write("set interfaces fxp0 unit 0 family inet address 10.0.0.15/24", "#")
-        self.wait_write("commit and-quit")
-        self.wait_write("exit", "commit complete")
+        # set interface fxp0 on dedicated management vrf, to avoid 10.0.0.0/24 to overlap with any "testing" network
+        self.wait_write("set system management-instance", "#")
+        self.wait_write("set routing-instances mgmt_junos description management-instance", "#")
+        # allow NATed outgoing traffic (set the default route on the management vrf)
+        self.wait_write("set routing-instances mgmt_junos routing-options static route 0.0.0.0/0 next-hop 10.0.0.2", "#")
```
Comment (hellt) [inline vsrx/docker/launch.py:109]: @RFC2516, @exhar
given you have played with vsrx recently, do you agree that these changes proposed by @ssasso?

Just wanted to get more than one +1 on this before we merge it.

Comment (RFC2516) [reply vsrx/docker/launch.py:109]: At face value I agree, however let me run my test script with the above bootstrap config and I will be able to give you a yes as well as a data point.

Comment (RFC2516) [reply vsrx/docker/launch.py:109]: Confirmed via test bench script from #118 that the launch success rate is greater than 95%.

+1 on this for merge.

Comment (hellt) [reply vsrx/docker/launch.py:109]: thank you @RFC2516 !
Do you have ideas why we have 5% fail rate?

Comment (RFC2516) [reply vsrx/docker/launch.py:109]: @hellt 

I am pondering the same thing. Let me modify my test script to preserve the state on failure.

**Pull request merged**